spec/support/generator_spec_helpers.rb

```diff
@@ -37,7 +37,7 @@ def each_file_in(path)
 
   def reset_routes
     Rails.application.routes.clear!
-    load "spec/example_app/config/routes.rb"
+    Rails.application.routes_reloader.execute
```

Comment:
Ooh, that's a nice change from just loading the file!

Reply:
I needed this since there were some active storage direct upload URLs that didn't get reset without it.

Reply:
Ahh, interesting!